File: string/171_ExcelSheetColumnNumber/171_ExcelSheetColumnNumber.py
class Solution:
    def titleToNumber(self, columnTitle: str) -> int:
        val = 0
        length = len(columnTitle)
        counter = length

        while counter > 0 :
            pos = length - counter
            ch = columnTitle[pos]
            val = val * 26 + ord(ch) -64
            counter = counter - 1
        
        return val

File: string/171_ExcelSheetColumnNumber/171_ExcelSheetColumnNumber.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
 var titleToNumber = function(columnTitle) {
    let length = columnTitle.length;    
    let counter = length;
    let val =  0;

    while(counter > 0){
        let pos = length - counter;
        val = val * 26 + (columnTitle.charCodeAt(pos) - 64);
        counter = counter -1;
    }
    
    return val;
};